docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -323,7 +349,7 @@ The latest version of the `Authenticator` has a completely different set of CSS
 Previous versions of `Authenticator` exposed a `onAuthUIStateChange` handler to detect Auth state changes. For similar functionality see [useAuthenticator](/react/connected-components/authenticator/advanced#access-auth-state).
 
 ## `@aws-amplify/ui-react-liveness`
-### 1.x to 2.x
+### Migrate from 1.x to 2.x
```

Comment:
Fixes an runtime error due to ids not being allowed to start with a number

![CleanShot 2023-10-13 at 17 05 34](https://github.com/aws-amplify/amplify-ui/assets/6165315/5e68e60b-cb57-4270-bda1-b007547d890d)